### ISSUE TITLE
vadOptions default value

### ISSUE THREAD

**Issue by ngcheeyuan** [SYSTRAN/faster-whisper#925]
Just putting it out here. It's a good idea to tune your VadOptions if you are facing some issues with it.

Working with non-english inputs,

The default value for VadOptions is a little to aggressive, and I personally recommend a longer padding and lower threshold values.

For padding, what I found was that it needed a longer padding because it tends to cut off the first few syllabus of the words if it was too aggressive.

**Comment by ooobo** [comment]
Be nice to hear what values have worked well for other users (for all languages) - maybe a discussion would be better?

**Comment by ckgithub2019** [comment]
> Just putting it out here. It's a good idea to tune your VadOptions if you are facing some issues with it.
> 
> Working with non-english inputs,
> 
> The default value for VadOptions is a little too aggressive, and I personally recommend a longer padding and lower threshold values.
> 
> For padding, what I found was that it needed a longer padding because it tends to cut off the first few syllabus of the words if it was too aggressive.

a longer padding means a longer "speech_pad_ms"? By default, the speech_pad_ms is 400, should be changed to 800 or even higher? what's your suggestion?

**Comment by ngcheeyuan** [comment]
> longer padding means a longer "speech_pad_ms"? By default, the speech_pad_ms is 400, should be changed to 800 or even higher?

Yes. I went with 1000ms and 0.4 threshold iirc. Do listen to the audio before and after silerovad is applied.
